src/site/content/en/lighthouse-performance/unused-javascript/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Remove unused JavaScript
+description: |
+  Learn how to pass Lighthouse's "Remove unused JavaScript" audit.
+web_lighthouse:
+  - unused-javascript
+date: 2020-06-17
+---
+
+Unused JavaScript can slow down your page speed if the JavaScript is
+[render-blocking][crp]. When the browser encounters a render-blocking script
+it has to download, parse, compile, and evaluate the script before it
+can proceed with all of the other work that's needed for rendering the page.
+
+{% Aside %}
+  Since the JavaScript code that Lighthouse flags in this audit is unused,
+  there is no evaluation cost. However, there are download, parse, and compilation
```

Comment:
I feel `parse and compilation` is normally just represented as `parse` time. Or is compilation part of execution? IDK, but it should match the `Reduce JavaScript execution time` categorization.

Reply:
*Reduce main thread work* references script parsing and compilation:

![image](https://user-images.githubusercontent.com/4713486/86850070-7e373a80-c065-11ea-9647-7a142966f4c5.png)